Mathlib/Combinatorics/SimpleGraph/Turan.lean

```diff
@@ -124,10 +124,12 @@ lemma exists_isTuranMaximal :
 
 end Defs
 
-namespace IsTuranMaximal
+section Forward
 
```

Comment:
I suspect this, and the corresponding `end Forward` at the end, must be a bad merge.